docs/release_notes/v1.10.0.md

```diff
@@ -0,0 +1,5 @@
+# Dapr 1.10.0
```

Comment:
The release notes are automated. The best way to track deprecations is to create a docs issue since we add those there for user reference. For now, I am OK to keep this here as a reminder for us when generating release notes for 1.10.